### PR TITLE
Feature/screen on off

### PR DESCRIPTION
**Pull request by cableman**
https://jira.itkdev.dk/browse/BIB-366

### CODE REVIEW

File: plugins/screen/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Screen plugin
+Turns the screen on/off base on configuration.
+
+# Events
+
+  * On
```
Comment (tuj) [inline plugins/screen/README.md]: ?

File: plugins/screen/screen.js [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+  // Build time in cron format from configuration format.
+  var on = config.on.split(':').reverse().join(' ');
+  var off = config.off.split(':').reverse().join(' ');
+
+  // Stop jobs if they are running.
+  this.stop();
+
+  // Set new jobs.
+  this.sceenOnjob = new ScreenCronJob(on + ' * * *', function() {
```
Comment (tuj) [inline plugins/screen/screen.js]: OnJob/OffJob instead of Onjob/Offjob

File: plugins/screen/screen.js [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+
+/**
+ * The Screen object.
+ *
+ * @param {object} bus
+ *   The event bus.
+ *
+ * @constructor
+ */
+var Screen = function Network(bus) {
```
Comment (tuj) [inline plugins/screen/screen.js]: Screen = Network ?

File: plugins/screen/screen.js [hunk 4]
```diff
@@ -0,0 +1,142 @@
+/**
+ * @file
+ * Checks if the application has online connection.
```
Comment (tuj) [inline plugins/screen/screen.js]: Copy/paste?

**Review by tuj — CHANGES_REQUESTED** [review]

File: plugins/screen/screen.js (hunk 4 above)
Comment (cableman) [reply plugins/screen/screen.js]: fixed

File: plugins/screen/screen.js (hunk 3 above)
Comment (cableman) [reply plugins/screen/screen.js]: fixed

File: plugins/screen/screen.js (hunk 2 above)
Comment (cableman) [reply plugins/screen/screen.js]: Fixed

File: plugins/screen/README.md (hunk 1 above)
Comment (cableman) [reply plugins/screen/README.md]: Fixed

**Review by tuj — APPROVED** [review]

**Pull request merged**